docs/docs/general/02-Components/AskUI-Development-Environment.md

```diff
@@ -185,3 +186,74 @@ Note: Adjust the parameters as needed for your specific configuration.
 :::
 
 See also the dedicated [AskUI Runner docs](AskUI-Runner.md) for more information.
+
+
+## ADE + VSCode IDE
+
+The ADE is build to work seamlessly with (VSCode)[https://code.visualstudio.com/] together. Therefore we create (VScode Workspace Settings)[https://code.visualstudio.com/docs/getstarted/settings#_settingsjson]
+
+**Note: Only compatible with windows**
+
+1. Create a `<project>/.vscode` folder with `mkdir .vscode`
+
+### Configure the askui-shell
+
+2. Create the `<project>/.vscode/settings.json` with following content:
+```json
+{
+    "livePreview.customExternalBrowser": "Default",
+    "terminal.integrated.profiles.windows":{   
+        "askui-shell": {    
+            "path": ["C:\\Program Files\\AskUI GmbH\\AskUI Suite\\Tools\\askui-shell.cmd"],
+            "args": [
+               "activate"
+            ],
+            "icon": "terminal-bash",
+        },        
+        "Git Bash": null,        
+        "Command Prompt": null,        
+        "PowerShell": null,        
+        "JavaScript Debug Terminal": null
+    },
+    "terminal.integrated.defaultProfile.windows": "askui-shell"
+}
+```
+- `terminal.integrated.profiles.windows`: Configures the `askui-shell` and deactivate other shells
```

Comment:
Have we explained to the user what a shell in general and the `askui-shell` is and what activating and deactivating and configuring shells means? If not, I think this may be a blocker for some users.

Reply:
I think you missing some context. Did you read the complete AskUI-Development-Environment.md? It's written in the first section. See https://github.com/askui/askui/blob/e4af00c2269e83f02b8d6d9abe94c04cd7dc9fd4/docs/docs/general/02-Components/AskUI-Development-Environment.md

Reply:
Yes, I read it but I could not find it explained to the user except that `askui-shell` makes askui commands available. But I think the user would like to know how the shell behaves other than that, e.g., are only askui commands available or is it like a powershell + askui commands? Why do we deactivate the other shells? Etc. But maybe I am wrong with that assumption. In doubt, let's go with version and check how users react.

Reply:
I think we should go with this version. A user _should_ have followed the Getting Started and know what the `askui-shell` is about.

You are right @adi-wan-askui that this is not always the case though! We should adapt this quickly once this comes up repeatably.